test/test-cases.setup.js

```diff
@@ -0,0 +1,40 @@
+import { basename, dirname, join } from 'path';
+import webpack from 'webpack';
+import I18nPlugin from '../src';
+
+export default function processFile(entry, ...pluginOpts) {
+  const resolvedEntry = join(__dirname, 'test-cases', entry);
+  const resolvedOutput = join(dirname(resolvedEntry), `${basename(resolvedEntry, '.code.js')}.tmp.js`);
+
+  const compiler = webpack({
+    entry: resolvedEntry,
+    output: {
+      filename: basename(resolvedOutput),
+      path: dirname(resolvedOutput),
+
+      libraryTarget: 'commonjs2',
```

Comment:
`output.libraryTarget` adds the global `module.exports` for the entire module, so it will be possible to require it. I didn't manage to achieve the similar behavior with the `target` option :(